src/sentry/static/sentry/app/views/releases/detail/shared/releaseIssues.jsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'react-emotion';
+
+import IssueList from 'app/components/issueList';
+import {Panel, PanelBody, PanelItem} from 'app/components/panels';
+import space from 'app/styles/space';
+import {t} from 'app/locale';
+
+export default class ReleaseIssues extends React.Component {
+  static propTypes = {
+    query: PropTypes.object.isRequired,
+    version: PropTypes.string.isRequired,
+    orgId: PropTypes.string.isRequired,
+    // Provided only in the project version of release issues
+    projectId: PropTypes.string,
+  };
+
+  getResolvedPath() {
+    const {version, orgId, projectId} = this.props;
+
+    return projectId
+      ? `/projects/${orgId}/${projectId}/releases/${encodeURIComponent(
+          version
+        )}/resolved/`
+      : `/organizations/${orgId}/releases/${encodeURIComponent(version)}/resolved/`;
+  }
+
+  getIssuesPath() {
+    const {orgId, projectId} = this.props;
+
+    return projectId
+      ? `/projects/${orgId}/${projectId}/issues/`
+      : `/organizations/${orgId}/issues/`;
+  }
```

Comment:
You could do a `basePath()` method that lets you contain the org/project URL differences.

```javascript
get basePath() {
  let {orgId, projectId} = this.props;
  return projectId ? `/projects/${orgId}/${projectId}/` : `/organizations/${orgId}/`;
}

get issuesPath() {
  return `${this.basePath}issues/`;
}
```
🏌️‍♀️

Reply:
Makes sense, but I think i'll just leave as-is :)